flash/text/classification/model.py

```diff
@@ -86,10 +87,11 @@ def __init__(
             learning_rate=learning_rate,
             multi_label=multi_label,
             output=output or LabelsOutput(multi_label=multi_label),
+            local_files_only=local_files_only,
         )
         self.enable_ort = enable_ort
         self.set_state(TransformersBackboneState(backbone))
-        self.model = self.backbones.get(backbone)(num_labels=num_classes)
+        self.model = self.backbones.get(backbone)(num_labels=num_classes, local_files_only=local_files_only)
```

Comment:
I think this wont work unless you have it in local catch which wnt happen if you are using a model for the first time

Reply:
So should I close this PR? @rohitgr7

Reply:
not yet. let's wait for @Borda

Reply:
well, that was that I trained it online and @SeanNaren changed it in Flash that the Tonenizer is part of the checkpoint so when you load it from the checkpoint it works offline but not for training from the ground...

Reply:
and it seems it was not even merged yet, I see the fix as hanging branch [fix/serialize_tokenizer](https://github.com/PyTorchLightning/lightning-flash/tree/fix/serialize_tokenizer)